### ISSUE TITLE
date internationalization, common formats

### ISSUE THREAD

**Issue by vlikin** [n3-charts/line-chart#401]
In the previous version v1 I used common date formats. I passed the the formatter according d3 documentation.   ticksFormat: '%m/%d' It stops working. It seems you preprocess settings filtering them.

**Comment by chaosmail** [comment]
Yep you are right, just saw this line is commented out in https://github.com/n3-charts/line-chart/blob/master/src/factories/Axis.ts#L233
@lorem--ipsum is this just a linting problem or something seriouis?

**Comment by chaosmail** [comment]
@vlikin we will fix this as soon as possible!

**Comment by lorem--ipsum** [comment]
The line is commented out because of a linting problem, however it shouldn't be a problem in this specific case since the `clone` method is called for the grid only.

**Comment by naduda** [comment]
It's an angularjs directive and i think right way create type like as angularjs filter. 
For example, type: 'date:dd.MM.yyyy:+0430'
